tests/Feature/Models/GoalTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Tests\Feature\Models;
+
+use App\Models\Goal;
+use App\Models\User;
+use Carbon\Carbon;
```

Comment:
`User` and `Carbon` are both unused in this file, could you please delete these imports?

Reply:
+